pom.xml

```diff
@@ -3,18 +3,23 @@
     <parent>
       <groupId>org.jenkins-ci.plugins</groupId>
       <artifactId>plugin</artifactId>
-      <version>1.396</version>
+      <version>3.57</version>
     </parent>
 
 	<artifactId>cmvc</artifactId>
 	<packaging>hpi</packaging>
 	<name>Jenkins CMVC Plug-in</name>
 	<version>0.5-SNAPSHOT</version>
 	<url>http://wiki.jenkins-ci.org/display/JENKINS/CMVC+Plugin</url>
+  
+  <properties>
```

Comment:
_nit_ not following the original spacing in the file.

Reply:
changed